native/src/modules/app/StateType.js

```diff
@@ -26,6 +26,7 @@ export type CategoryRouteConfigType = {|
 |}
 
 export type CategoryRouteStateType = {|
+  +routeType: 'category',
```

Comment:
It think it would be nicer if we use a constant for this or an enum for routeTypes. maybe the constants from routes/index.js would make sense?

Reply:
This is a type, so not sure whether we can reuse that constant.

Maybe `typeof 'catgory'` would work. Maybe that typeof is string though.